src/config/fix_inversion.json

```diff
@@ -207,6 +215,10 @@
             "url": "reddit.com",
             "invert": "#header-img"
         },
+        {
+            "url": "redmine",
```

Comment:
Did you mean `redmine.org`?

Reply:
Hi,

Actually no, redmine is often (and also in my case) used in the internal network of companies, so their address is usually just redmine/
Does it make it too open? If this is bad for merging, is there a way for me to apply that only to me, directly in the extension? I have it in the Inversion Fix Editor, but every now and then I have to open it and apply again, because it returns to default.

Thanks for the great extension you made ;)

Reply:
I plan to improve the Editor during Christmas holidays, so that you will be able to use fixes for local sites and prevent them from being overriden by GitHub config. Currently I will only merge fixes for global web sites.

Reply:
Ok, cool, thank you!